src/Sulu/Bundle/AdminBundle/Resources/js/components/Input/input.scss

```diff
@@ -33,12 +33,21 @@ $inputFontSize: 12px;
     }
 }
 
+.prepended-container {
+    position: relative;
+    width: 34px;
```

Comment:
Do we have now a 34px gap on the left side of the input, if there is no icon?

Reply:
No, there is a jsx condition to prevent that